build.gradle.kts

```diff
@@ -207,14 +209,11 @@ allprojects {
         isFailOnError = false
     }
 
-    afterEvaluate {
-        // Note: plugins should override this
-        version = getWPILibVersion() ?: getVersionFromGitTag()
-        tasks.withType<Jar> {
-            manifest {
-                attributes["Implementation-Version"] = project.version as String
-                attributes["Built-Date"] = Instant.now().toString()
-            }
+    // Note: plugins should override this
```

Comment:
I'd move this note up to the `version = projectVersion`, since that's where the project's version is being set